templates/viewprofile.html

```diff
@@ -81,15 +81,15 @@ <h3 class="contact-center" >Contact Me</h3>
             <div class="panel panel-default">
                 <div class="panel-body">
                     <h3>Bio</h3>
-                    <p class="staff-profile staff-bio">{{ staff.bio }}
+                    <p class="staff-profile staff-bio">{{ staff.bio|safe }}
```

Comment:
It seems [`div` isn't allowed inside `p`](http://stackoverflow.com/questions/10763780/putting-div-inside-p-is-adding-an-extra-p/10763952#10763952) I get messed up html after editing a profile with the new editor.
Changing this to `<div>{{ staff.bio|safe }}</div>` makes it render correctly.